src/types/TodosContext.ts

```diff
@@ -0,0 +1,12 @@
+import { Todo } from './Todo';
+
+export type TodoContext = {
+  todos: Todo[]
+  setTodos: (v: Todo[] | ((n: Todo[]) => Todo[])) => void
```

Comment:
try to update to 
`setTodos: React.Dispatch<React.SetStateAction<Todo[]>>`

it has to work